### ISSUE TITLE
[Feature] Add Hold Item Descriptions

### ISSUE THREAD

**Issue by DaltJM** [pagefaultgames/pokerogue#2426]
**Is your feature request related to a problem? Please describe.**
Adding a description of what a hold items effect is, while a Pokemon is holding an item, like Silk Scarf, Dragon Fang etc instead of needing to look outside the game to confirm what they do.

**Describe the Feature**
Add the ability to view the description of a Pokemon's held item, either from the Party summary screen or the individual Pokemon's summary screen. This would be useful to reduce the need to check what an item does using Google.

**Comment by EmberCM** [comment]
You can already do this by hovering over the item with your mouse if you're on PC
![battle](https://i.ember.zone/9cxu4nIyDZqtNaUM.png)
![summary](https://i.ember.zone/DYoWpuK7DRhYlTPO.png)

**Comment by DaltJM** [comment]
True, but if my pick-up slave picks up a held item I can't check what it does unless I put it into battle. Also that won't work on mobile unfortunately.

**Comment by RNGezeus** [comment]
> True, but if my pick-up slave picks up a held item I can't check what it does unless I put it into battle. Also that won't work on mobile unfortunately.

You can already hover items on individual summary on PC in same manner.

**Comment by DaltJM** [comment]
So it sounds like this is more of a mobile/PC controller support suggestion.

**Comment by flx-sta** [comment]
@DaltJM can you please update the title accordingly to reflect the actual feature request better
